components/prism-editorconfig.js

```diff
@@ -0,0 +1,20 @@
+Prism.languages.editorconfig = {
+	// https://editorconfig-specification.readthedocs.io/en/latest/
+	'comment': /[;#].*$/m,
+	'section': {
+		pattern: /^[ \t]*\[.+]/m,
+		alias: 'keyword',
+		inside: {
+			'regex': /\\\\[\[\]{},!?.*]/, // Escape special characters with '\\'
+			'punctuation': /[\[\]{},!?]|\.\.|\*{1,2}/
```

Comment:
I think it would great if the `*`, `**`, `?`, `!`, and `..` were visually distinct from the rest.

```suggestion
			'operator': /[!?]|\.\.|\*{1,2}/,
			'punctuation': /[\[\]{},]/
```

Reply:
Yes, it looks much better